### PR TITLE
fix: wait for logger thread to quit gracefully before closing segger …

### PR DESCRIPTION
**Pull request by Pog3k**
fix segger thread closing. Wait gracefully for logging thread to quit.

Auto changelog tool seems to be unreliable. Shall be fixed in another task.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,13 +23,16 @@ Generator command: `npm run change`
...
+#### Fixes
+
+- fix: wait for logger thread to quit gracefully before closing segger channel [`039d662`](https://github.com/eclipse/kiso-testing/commit/039d6627f0e60948bbe1dc0a461e9203b50bd251)
 
-#### Docs
+---
+## [0.15.1](https://github.com/eclipse/kiso-testing/compare/0.15.0...0.15.1)
+> 2022-02-17
 
-- docs: add links for how to create an auxiliary and connector [`6da8315`](https://github.com/eclipse/kiso-testing/commit/6da83154c6c8d0653d87ae69cceb85d4694d2449)
```
Comment (Pog3k) [inline CHANGELOG.md:30]: changelog tool broken.. not sure what the issue is.. will fix in another task..

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]

**Comment by sebastianpfischer** [comment]
Why is there a tag issue? Can you somehow revert it manually and try to resolve the issue in another PR / tikcet?

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,13 +23,16 @@ Generator command: `npm run change`
 Commit helper: `cz c`
 
 
-## [0.15.1](https://github.com/eclipse/kiso-testing/compare/0.15.0...0.15.1)
+## [0.15.0](https://github.com/eclipse/kiso-testing/compare/0.15.1...0.15.0)
```
[the remark this answers is not in the document]
Comment (Pog3k) [reply CHANGELOG.md:26]: as mentioned
-> Auto changelog tool seems to be unreliable. Shall be fixed in another task.

File: src/pykiso/lib/connectors/cc_rtt_segger.py [hunk 3]
```diff
@@ -117,7 +117,7 @@ def __init__(
         self.rx_buffer_size = None
         # initialize rtt logging specific parameters
         self.rtt_configured = False
-        self._is_running = False
+        self._is_log_thread_running = False
```
Comment (sebclrsn) [inline src/pykiso/lib/connectors/cc_rtt_segger.py:120]: Maybe just _logger_thread_running would be enough

**Review by sebclrsn — APPROVED** [review]

File: src/pykiso/lib/connectors/cc_rtt_segger.py (hunk 3 above)
Comment (Pog3k) [reply src/pykiso/lib/connectors/cc_rtt_segger.py]: as you whish.. 
done

**Pull request merged**